src/common/helper/url-share.ts

```diff
@@ -1,4 +1,5 @@
 import defaults from "../constants/defaults.json";
+import auth from "../pages/auth";
 
 export const makeEsteemUrl = (cat: string, author: string, permlink: string): string =>
```

Comment:
change naming to `makeEcencyUrl` instead

Reply:
@feruzm  updated.